packages/relay/src/lib/eth.ts

```diff
@@ -326,10 +328,9 @@ export class EthImpl implements Eth {
     requestDetails: RequestDetails,
   ): Promise<IFeeHistory | JsonRpcError> {
     const requestIdPrefix = requestDetails.formattedRequestId;
-    const maxResults =
-      ConfigService.get('TEST')
-        ? constants.DEFAULT_FEE_HISTORY_MAX_RESULTS
-        : Number(ConfigService.get('FEE_HISTORY_MAX_RESULTS'));
+    const maxResults = ConfigService.get('TEST')
```

Comment:
nit: for a different PR this feels like something the ConfigService should handle internally.
Is that possible?

Reply:
Yes, we could add a `defaultValue` for `FEE_HISTORY_MAX_RESULTS` in the **globalConfig.ts** file

Reply:
Opened an issue for that https://github.com/hashgraph/hedera-json-rpc-relay/issues/3176